packages/hash/api/src/graphql/resolvers/user/joinOrg.ts

```diff
@@ -28,13 +28,14 @@ export const joinOrg: ResolverFn<
 
     const { invitationLinkToken, invitationEmailToken } = verification;
 
+    /** @todo: potentially deprecate these method calls depending on the org invitation implementation */
     const invitation = invitationLinkToken
-      ? await org.getInvitationLinkWithToken(client, {
+      ? await (org as any).getInvitationLinkWithToken(client, {
```

Comment:
To get linting to pass. We haven't implemented `getInvitationLinkWithToken`, and I don't think we're going to given I prior discussion I had with David with how we will rework org invitations. I'm not comfortable deleting it because this PR isn't replacing the intended functionality here. I think this will be addressed as part of [this task](https://app.asana.com/0/1202805690238892/1202980861294704/f) (which is now linked in the todo a couple lines up in this file)

Reply:
Gotcha. Can you add a lint then? 

I would prefer to throw an error saying "This is uninimplemented" or something rather than just hiding it with an `as any` cast though (even if that requires to comment out code)

Reply:
> Can you add a lint then?

Sorry, what do you mean? Add a `@ts-ignore`?

Reply:
Sorry I meant comment, my brain is at half capacity. Can you add a comment / TODO*

Reply:
> even if that requires to comment out code to avoid compilation errors

Commenting this out would propagate linting errors throughout the file, so I'd have to comment out around 100-200 lines of code. Is that preferable to just adding a `@ts-ignore` for now?

Reply:
There's just nothing in this file to tell me that it's broken. I'd almost prefer us to comment out the entire file or something if we're not even sure we're going to keep it. 

It wouldn't be obvious to remove these `as any` casts if we did try and fix this file.

Maybe we should just throw a new Error saying it's unimplemented at the top, and then comment out the rest of the resolver.

Or add a `ts-ignore` like you're suggesting, but below that error, with a comment explaining it.

Reply:
Turns out I can't add `ts-ignore` because it creates an eslint error 😢. To avoid this mess I'm just going to uncomment the implementation as you're suggesting, it just makes it harder to read prior `@todo`s that were left in that implementation ([such as this one](https://github.com/hashintel/hash/blob/8238cad7e65bef6dd5d43aef599a24ab1ebf5ea4/packages/hash/api/src/graphql/resolvers/user/joinOrg.ts#L31-L34))

Reply:
> Maybe we should just throw a new Error saying it's unimplemented at the top, and then comment out the rest of the resolver.

Done in https://github.com/hashintel/hash/pull/1068/commits/2994caca7a9a7921d189722eb088a0bb312feb82